grants_tagger_light/training/train.py

```diff
@@ -108,4 +108,21 @@ def train_bertmesh_cli(
 
 
 if __name__ == "__main__":
-    train_app()
+    from dataclasses import dataclass
+
+    @dataclass
+    class TrainFuncArgs:
```

Comment:
why is that needed and we cannot trigger the `train_app`?

Reply:
That could be possible but then we'd need to run this from dvc, and I don't know if `typer` works the same when invoking directly as a script.

Reply:
Just tested and it won't detect the `ctx` with the extra args. I would keep it like this for now.